pandas/tests/io/formats/test_format.py

```diff
@@ -1434,6 +1434,13 @@ def test_repr_html(self):
 
         tm.reset_display_options()
 
+    def test_repr_html_mathjax(self):
```

Comment:
can you add a test for Series as well

Reply:
We don't have a repr_html for series yet.

Reply:
ok then!